Content.Shared/Silicons/StationAi/SharedStationAiSystem.Held.cs

```diff
@@ -62,9 +68,11 @@ private void OnCoreJump(Entity<StationAiHeldComponent> ent, ref JumpToCoreEvent
 
     private void OnToggleBolts(Entity<StationAiHeldComponent> ent, ref AiToggleBoltsEvent args)
     {
-        if (!TryGetCore(ent.Owner, out var core) || core.Comp?.RemoteEntity == null)
+        if (!TryGetCore(ent.Owner, out var core) || core.Comp?.RemoteEntity == null || args.Handled)
```

Comment:
```suggestion
        if (args.Handled || !TryGetCore(ent.Owner, out var core) || core.Comp?.RemoteEntity == null)
```
Буловые переменные лучше проверять первыми, чтобы в случае если их значение true, остальные проверки не выполнялись